CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.4.0] - 2017-06-20
+
+### Added
+
+- Support publisher confirms.
+
+### Changed
+
+- Improve main README.
+- Improve RPC README.
```

Comment:
I don't think improving the readme actually deserves being part of the changelog. It does not change the library.